rust/Cargo.toml

```diff
@@ -1,5 +1,5 @@
 [package]
-name = "chainblocks"
+name = "chainblocks-rust"
 description = "rust bindings and utility in order to use chainblocks"
```

Comment:
This is wrong, actually, this includes also blocks and many now fundamental components